packages/document/docs/en/guide/faq/hmr.md

```diff
@@ -56,7 +56,7 @@ export default {
 };
 ```
 
-To solve this problem, you need to reference the development builds of React or not configure `externals` in the development environment.
```

Comment:
why React Devtools needed?

Reply:
cdn import must need this, detail principle can take a look of my sharing